Kitodo/src/main/java/org/kitodo/production/services/data/ProjectService.java

```diff
@@ -249,7 +249,7 @@ public boolean isProjectComplete(Project project) {
         boolean projectsXmlExists = KitodoConfigFile.PROJECT_CONFIGURATION.exists();
         boolean digitalCollectionsXmlExists = KitodoConfigFile.DIGITAL_COLLECTIONS.exists();
 
-        return Objects.nonNull(project.getTitle()) && Objects.nonNull(project.template)
+        return Objects.nonNull(project.getTitle())
```

Comment:
Why did you remove this?

Reply:
The only use of the `project.template` field was to list in the web service which production templates were assigned to the project. Since the web service is removed, it is no longer needed.